alerta/auth/oidc.py

```diff
@@ -35,7 +35,7 @@ def get_oidc_configuration(app):
     except Exception as e:
         raise ApiError('Could not get OpenID configuration from well known URL: {}'.format(str(e)), 503)
 
-    if config['issuer'] != issuer_url:
+    if config['issuer'].format(tenantid=app.config['AZURE_TENANT']) != issuer_url:
```

Comment:
This fixes the issuer URL verification for Azure which returns literally returns `{tenant}` in the issuer URL.